packages/flutter_tools/test/web.shard/expression_evaluation_web_ddc_library_bundle_test.dart

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+@TestOn('linux') // https://github.com/flutter/flutter/issues/169304
 @Tags(<String>['flutter-test-driver'])
-@Skip('https://github.com/flutter/flutter/issues/169304')
```

Comment:
Should we unskip [this one](https://github.com/flutter/flutter/blob/9c89a8fff9e6eb0a1065f99dafeb1c8a172a4d16/packages/flutter_tools/test/web.shard/expression_evaluation_web_amd_test.dart#L6) too?